deps/v8/src/debug/debug.cc

```diff
@@ -3544,6 +3555,17 @@ void FunctionCallbackRecordReplayCurrentExecutionPoint(const FunctionCallbackInf
   std::string point = RecordReplayGetCurrentExecutionPoint();
   i::Handle<i::String> rv = CStringToHandle(isolate, point.c_str());
   args.GetReturnValue().Set(Utils::ToLocal(rv));
+
+  if (!gHasNewCheckpointFlushedTask) {
+    gHasNewCheckpointFlushedTask = true;
+
+    auto task_runner = i::V8::GetCurrentPlatform()->GetForegroundTaskRunner(args.GetIsolate());
+    auto task = i::MakeCancelableTask(isolate, []() {
```

Comment:
Does the `Cancelable` indicate anything we need to be worried about? Worried what would happen if this callback never runs and we're stuck with `gHasNewCheckpointFlushedTask == true` forever.

Reply:
I don't think so, we'd need to explicitly cancel the task which we don't do.  Having a way to post a std::function that isn't a cancelable task would be nicer, but I didn't find anything like that with some cursory searching through v8.